src/BrightScriptCommands.ts

```diff
@@ -55,6 +56,13 @@ export class BrightScriptCommands {
             this.activeDeviceManager.refresh();
         });
 
+        // Register the new project creation command
+        this.context.subscriptions.push(
```

Comment:
This is doing the same work as down below in line 347, so you can delete this. It actually causes errors in vscode to double-register the same command.